src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -221,7 +221,10 @@ class EventEntries extends React.Component<Props> {
         )}
         {!isShare &&
           (showExampleCommit ? (
-            <EventCauseEmpty organization={organization} project={project} />
+            <EventCauseEmpty
+              organization={organization as Organization}
```

Comment:
Is there a way we can refactor this so that we type guard `organization` here? e.g. organization is only `Organization` when `isShare` is false. I could see someone refactoring this so that it's not protected by the `!isShare` condition which could cause issues.

Reply:
Yeah I can update the isShare conditions.